### PR TITLE
update javafx and gluonfx plugins

### PR DESCRIPTION
**Pull request by abhinayagarwal**
updates plugin to latest version

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,19 +1,22 @@
...
-    id 'com.gluonhq.gluonfx-gradle-plugin' version '1.0.1' apply false
+    id 'org.openjfx.javafxplugin' version '0.1.0' // Client plugin goes berserk is apply is false here
+    id 'com.gluonhq.gluonfx-gradle-plugin' version '1.0.21-SNAPSHOT' apply false
 }
 
 ext.libraryVersions = [
         attach: '4.0.10',
         glisten: '6.0.5',
-        javafx : '15.0.1'
+        javafx : '21'
```
Comment (jperedadnr) [inline build.gradle:10]: Can you update Attach and Glisten versions too?

File: build.gradle [hunk 2]
```diff
@@ -1,19 +1,22 @@
...
 ]
 
 subprojects {
     apply plugin: 'application'
-    apply plugin: 'org.openjfx.javafxplugin'
     apply plugin: 'com.gluonhq.gluonfx-gradle-plugin'
+
+    configure([project(':HelloFX'), project(':HelloFXML'), project(':HelloGluon')]) {
+        apply plugin: 'org.openjfx.javafxplugin'
```
Comment (jperedadnr) [inline build.gradle:18]: Alternatively, remove HelloWorld?

File: settings.gradle [hunk 3]
```diff
@@ -1,5 +1,6 @@
 pluginManagement {
     repositories {
+        mavenLocal()
```
Comment (jperedadnr) [inline settings.gradle]: sonatype?

Comment (abhinayagarwal) [reply settings.gradle]: `mavenCentral()` ?

Comment (jperedadnr) [reply settings.gradle]: I was referring to sonatype, as there was the snapshot. With the release, no change is required.

**Review by jperedadnr — APPROVED** [review]

**Pull request merged**